core/storage/feedback/gae_models.py

```diff
@@ -304,7 +304,26 @@ class UnsentFeedbackEmailModel(base_models.BaseModel):
     # Each element in this list is a dict with keys 'exploration_id',
     # 'thread_id' and 'message_id'; this information is used to retrieve
     # corresponding FeedbackMessageModel instance.
-    feedback_messages = ndb.JsonProperty(repeated=True)
+    feedback_message_references = ndb.JsonProperty(repeated=True)
     # The number of failed attempts that have been made (so far) to
     # send an email to this user.
     retries = ndb.IntegerProperty(default=0, required=True, indexed=True)
+
+    @classmethod
+    def create_or_modify(cls, owner_id, feedback_message_reference):
+        """Creates a new instance of UnsentFeedbackEmailModel for sending
+        feedback message email"""
+
+        model = cls.get(owner_id, strict=False)
+        new_instance = True
+
+        if model:
+            model.feedback_message_references.append(feedback_message_reference)
+            model.put()
+            return not new_instance
```

Comment:
I think just "return False" here is fine, and ditto below. No need for the extra variable; better to clarify in the docstring.